build.gradle

```diff
@@ -12,6 +13,7 @@ buildscript {
         classpath "com.netflix.nebula:gradle-lint-plugin:10.1.3"
         classpath "de.aaschmid:gradle-cpd-plugin:1.1"
         classpath "io.codearte.gradle.nexus:gradle-nexus-staging-plugin:0.12.0"
```

Comment:
Should `gradle-nexus-staging-plugin` not be removed?

Reply:
No, see my answer above